src/org/zaproxy/zap/extension/pscan/PassiveScanParam.java

```diff
@@ -155,4 +165,37 @@ public void setScanOnlyInScope(boolean scanOnlyInScope) {
     public boolean isScanOnlyInScope() {
         return scanOnlyInScope;
     }
+    
+    /**
+     * Sets whether or not the passive scan should be performed on traffic
+     * from the fuzzer.
+     *
+     * @param scanFuzzerMessages
+     *            {@code true} if the scan should be performed on traffic
+     *            generated by the fuzzer, {@code false} otherwise.
+     * @since TODO Add version
+     * @see #isScanFuzzerMessages()
+     */
+    public void setScanFuzzerMessages(boolean scanFuzzerMessages) {
+        this.scanFuzzerMessages = scanFuzzerMessages;
+        getConfig().setProperty(SCAN_FUZZER_MESSAGES_KEY, scanFuzzerMessages);
+        if (scanFuzzerMessages) {
```

Comment:
Just noticed that with this change we now need to execute this logic after reading from the file. (Extract a method and call from both places?)

Reply:
Sure, is it he file read it n this class?

Reply:
What file?

Reply:
Sorry, I was referring to the `parse` method.

Reply:
Thanks for clarifying, will look into it.